mappings/net/minecraft/client/gui/Gui.mapping

```diff
@@ -22,6 +22,7 @@ CLASS none/bgn net/minecraft/client/gui/Gui
 		ARG 0 mouseX
 		ARG 1 mouseY
 		ARG 2 mouseButton
+	METHOD a mouseDragged (IIIJ)V
```

Comment:
I think we should try to be consistent with our own mappings over those of Mojangs's. Especially since we don't have enough of their mappings to go from to make a conclusive picture.